source/content.css

```diff
@@ -481,6 +481,11 @@ td.js-line-comments:last-child .timeline-comment-actions .add-reaction-popover::
 }
 
 /* +/- Pseudo elements on diffs */
+.refined-github-diff-signs .blob-code-addition,
+.refined-github-diff-signs .blob-code-deletion {
```

Comment:
Could also think of simplifying it to `.refined-github-diff-signs .blob-code`. WDYT?

Reply:
No.

Reason: #613 #612 #607 #601 #590 #563 #557 #396 #382 #317

In short: this feature is very fickle and I don't want to touch it unless it's broken.